app/views/inthenews/news.html.erb

```diff
@@ -0,0 +1,78 @@
+<h1 class="sectiontitle"><%= t "inthenews.title" %></h1>
+
+<%= render "inthenews/top_include" %>
+
+<h2>News</h2>
+
+<div class="row">
+  <div class="col-md-5">
+    
+    <label for="exampleInput">Search News</label>
+    <div class="input-group">
+      <input type="text" class="form-control" placeholder="Search for...">
+      <span class="input-group-btn">
+        <button class="btn btn-default" type="button">Go!</button>
+      </span>
+    </div><!-- /input-group -->
+
+    <p><br/>Select a news format to see the distribution of poetry news around the continent. Search by decade by sliding the timeline to see the distribution of poetry news publications in regions of Africa during that time period.</p>
+
+    <!-- <div class="btn-group">
```

Comment:
can this be deleted?

Reply:
I would like to leave it because I'm not clear if lorna wants this here on in the panel on the right. I'll clean up more later! (I did change the comment to rails style so it won't show in source